docs/copilot/customization/language-models.md

```diff
@@ -70,57 +68,62 @@ To change the language model that is used for generating code completions in the
 
 1. Select **Change Completions Model...**, and then select one of the models from the list.
 
+> [!NOTE]
+> The models that are available for code completions might evolve over time as we add support for more models.
+
 ## Bring your own language model key
 
 > [!IMPORTANT]
 > This feature is not currently available to Copilot Business or Copilot Enterprise users.
 
-If you already have an API key for a language model provider, you can use their models in chat in VS Code, in addition to the built-in models that Copilot provides. You can use models from the following providers: Anthropic, Azure, Google Gemini, Groq, Ollama, OpenAI, OpenRouter, and xAI.
+GitHub Copilot in VS Code comes with a variety of built-in language models that are optimized for different tasks. If you want to use a model that is not available as a built-in model, you can bring your own language model API key (BYOK) to use models from other providers.
 
-Using your own language model API key in VS Code has several advantages:
+VS Code has built-in support for the following model providers: Anthropic, Azure, Google, Groq, Ollama, OpenAI, OpenRouter, and xAI. If you have a model that is compatible with the OpenAI API, you can also [configure a custom OpenAI-compatible model](#use-an-openai-compatible-model).
 
-- **Model choice**: access hundreds of models from different providers, beyond the built-in models.
-- **Experimentation**: experiment with new models or features that are not yet available in the built-in models.
-- **Local compute**: use your own compute for one of the models already supported in GitHub Copilot or to run models not yet available.
-- **Greater control**: by using your own key, you can bypass the standard rate limits and restrictions imposed on the built-in models.
+Using your own language model API key in VS Code has several benefits:
 
-To manage the available models for chat:
+* **Model choice**: access hundreds of models from different providers, beyond the built-in models.
+* **Experimentation**: experiment with new models or features that are not yet available in the built-in models.
+* **Local compute**: use your own compute for one of the models already supported in GitHub Copilot or to run models not yet available.
+* **Greater control**: by using your own key, you can bypass the standard rate limits and restrictions imposed on the built-in models.
 
-1. Select **Manage Models** from the language model picker in the Chat view.
+You can add more model providers by installing extensions from the [Visual Studio Marketplace](https://marketplace.visualstudio.com/vscode), such as [AI Toolkit for VS Code](https://aka.ms/AIToolkit), [Cerebras Inference](https://aka.ms/vscode/cerebras), [Hugging Face](https://aka.ms/vscode/huggingface), and others.
```

Comment:
Let's use this link somewhere https://marketplace.visualstudio.com/search?term=tag%3Alanguage-models&target=VSCode&category=All%20categories&sortBy=Relevance

Since that is listing all extensions that publish langauge models - right now the list is only 1 extension but it will grow over time.